index.js

```diff
@@ -117,6 +131,38 @@ module.exports = function (cmd, args, opts) {
 		handleInput(spawned, parsed.opts);
 	});
 
+	if (opts && (opts.stdout === 'observable' || (opts.stdout && opts.stdout.observable))) {
```

Comment:
The options handling here is not well thought out. Just wanted to get something working.